client/simple-example.py

```diff
@@ -0,0 +1,13 @@
+import flaschen
+
+UDP_IP = 'localhost'
+UDP_PORT = 1337
+
+ft = flaschen.Flaschen(UDP_IP, UDP_PORT, 45, 35)
+
+while True:
+  for b in xrange(0, 256):
+    for y in xrange(0, 35):
```

Comment:
nitpick: using ft.width and ft.height in this loop might be nicer so that the same value only has to be set in the constructor above, not in two places in the code.